docs/examples/dcgan.md

```diff
@@ -44,7 +44,7 @@ $ floyd run \
 --gpu \
 --env pytorch-0.2 \
 --data redeipirati/datasets/lfw/1:lfw \
-"python main.py --dataset lfw --dataroot /lfw --outf /output --cuda --ngpu 1 --niter 20"
+"python main.py --dataset lfw --dataroot /lfw --outf trained-models --cuda --ngpu 1 --niter 20"
```

Comment:
Nit: use `trained_models`. Using dash makes it look like a parameter name.

Applies to few other places below.